app/views/talents/edit.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="container">
+  <div class="row mt-20 mb-20">
+    <div class="col-xs-6 col-sm-6 col-md-6 col-lg-6">
+      <h2>Edit Talent</h2>
+    </div>
+  </div>
+  <hr>
+  <%= render 'form', :talent => @talent %>
```

Comment:
In general it's better to use the new syntax (colon syntax):

```ruby
render 'form', talent: @talent
```

Instead of the "hashrocket" syntax

```
render 'form', :talent => @talent
```

Hashrock syntax was commonly used prior ruby 1.9